include/deal.II/lac/lapack_support.h

```diff
@@ -26,7 +26,7 @@ DEAL_II_NAMESPACE_OPEN
 namespace LAPACKSupport
 {
   /**
-   * Most LAPACK functions change the contents of the matrix applied to to
```

Comment:
Ah, yes, now I see. This just look like a typical typo. Maybe reformatting ist still not a bad idea.